app/src/main/java/com/framgia/englishconversation/screen/createPost/CreatePostViewModel.java

```diff
@@ -520,7 +538,7 @@ private void onAudioRecordingClick() {
                 new RecordingAudioDialog.OnRecordingAudioListener() {
                     @Override
                     public void onRecordingAudioClick(String filePath, String fileName) {
-                        releaseExoAudioPlayer();
+                        storeDataWhenReleasePlayer();
```

Comment:
ERROR: 'method def' child have incorrect indentation level 24, expected level should be one of the following: 16, 20.